integrations/access/opsgenie/client.go

```diff
@@ -135,14 +136,19 @@ func NewClient(conf ClientConfig) (*Client, error) {
 }
 
 func errWrapper(statusCode int, body string) error {
```

Comment:
Nit: Modify function parameter to accept `body` as `[]byte`. The `resty.Response` returns the body as `[]byte`. No need to convert to `string` and then back to `[]byte`.